install/kubernetes/helm/istio/charts/kiali/values.yaml

```diff
@@ -3,8 +3,8 @@
 #
 enabled: false # Note that if using the demo or demo-auth yaml when installing via Helm, this default will be `true`.
 replicaCount: 1
-hub: docker.io/kiali
-tag: v0.16
+hub: quay.io/kiali
```

Comment:
can you offer some guidance on the permanent location for "new" images for kiali?  Clearly, we don't want to delete docker.io/kiali as it would damage our 1.1 releases in the field.  The question then becomes, will docker.io/kiali become a mirror, or will docker.io/kiali become deprecated (starting with this PR or has it already?)

As to using `quay.io/kiali` as the canonical source, that seems fine, but I'd like some clarity on how the images will be managed.  Will docker.io/kiali only contain old versions?

It is highly atypical that a hub would change anywhere across the cloud native ecosystem for an emerging or mature project and Istio is no exception to this rule.  Still, willing to work with the Kiali community if this is Kiali's desired path forward.

Can you please add clarity here?

Reply:
> Clearly, we don't want to delete docker.io/kiali as it would damage our 1.1 releases in the field.

The docker.io repository for Kiali is still there and will retain the old versions that are now currently. They will not be going away.

> will docker.io/kiali become a mirror, or will docker.io/kiali become deprecated (starting with this PR or has it already?)

Right now, docker.io is a mirror for us - we are now pushing Kiali images to BOTH docker.io and quay.io so as not to disrupt current users that may expect to still see Kiali images on docker.io - so you will still see Kiali images published to both. However, in the future, the plan is for Kiali to publish to quay.io as its canonical source. There are no plans right now for Kiali to stop publishing new images to docker.io, but we do want to start migrating our users over to using quay.io. And, again, old images will not be removed from docker.io regardless of this migration over to quay.io.